examples/hosts/app-integration/schema-upgrade/src/migrator/migrator.ts

```diff
@@ -86,100 +108,133 @@ export class Migrator extends TypedEventEmitter<IMigratorEvents> implements IMig
         }
 
         const doTheMigration = async () => {
-            // It's possible that our modelLoader is older and doesn't understand the new acceptedVersion.  Currently
-            // this fails the migration gracefully and emits an event so the app developer can know they're stuck.
-            // Ideally the app developer would find a way to acquire a new ModelLoader and move forward, or at least
-            // advise the end user to refresh the page or something.
-            // TODO: Does the app developer have everything they need to dispose gracefully when recovering with a new
-            // ModelLoader?
-            const migrationSupported = await this.modelLoader.supportsVersion(acceptedVersion);
-            if (!migrationSupported) {
-                this.emit("migrationNotSupported", acceptedVersion);
-                this._migrationP = undefined;
-                return;
-            }
+            // doTheMigration() is called at the start of migration and should only resolve in two cases. First, is if
+            // we complete the migration. Second, is if we disconnect during the migration process. In both cases we
```

Comment:
"Complete" means do actual migration or exit without completing because another client completed it ?

Reply:
It can mean either in this case. It just means that we exited after either the local client or another client has successfully completed the client. I can update the comment to clarify!

Reply:
Feel free to resolve this comments as you make clarifications.